src/main/java/org/isf/medicalinventory/gui/InventoryWardEdit.java

```diff
@@ -0,0 +1,1175 @@
+            panelHeader.add(getAllRadio(), gbc_allRadio);
+            GridBagConstraints gbc_searchTextField = new GridBagConstraints();
+            gbc_searchTextField.insets = new Insets(0, 0, 0, 5);
+            gbc_searchTextField.fill = GridBagConstraints.HORIZONTAL;
+            gbc_searchTextField.gridx = 3;
+            gbc_searchTextField.gridy = 2;
+            panelHeader.add(getSearchTextField(), gbc_searchTextField);
+            ButtonGroup group = new ButtonGroup();
+            group.add(specificRadio);
+            group.add(allRadio);
+
+        }
+        return panelHeader;
+    }
+
+    private JPanel getPanelContent() {
+        if (panelContent == null) {
+            panelContent = new JPanel();
+            GridBagLayout gbl_panelContent = new GridBagLayout();
+            gbl_panelContent.columnWidths = new int[] { 452, 0 };
+            gbl_panelContent.rowHeights = new int[] { 402, 0 };
+            gbl_panelContent.columnWeights = new double[] { 1.0, Double.MIN_VALUE };
+            gbl_panelContent.rowWeights = new double[] { 1.0, Double.MIN_VALUE };
+            panelContent.setLayout(gbl_panelContent);
+            GridBagConstraints gbc_scrollPaneInventory = new GridBagConstraints();
+            gbc_scrollPaneInventory.fill = GridBagConstraints.BOTH;
+            gbc_scrollPaneInventory.gridx = 0;
+            gbc_scrollPaneInventory.gridy = 0;
+            panelContent.add(getScrollPaneInventory(), gbc_scrollPaneInventory);
+        }
+        return panelContent;
+    }
+
+    private JPanel getPanelFooter() {
+        if (panelFooter == null) {
+            panelFooter = new JPanel();
+            panelFooter.add(getSaveButton());
+            panelFooter.add(getValidateButton());
+            panelFooter.add(getCancelButton());
+            panelFooter.add(getPrintButton());
+            panelFooter.add(getCloseButton());
+        }
+        return panelFooter;
+    }
+
+    private GoodDateChooser getJCalendarFrom() {
+        if (jCalendarInventory == null) {
+
+            jCalendarInventory = new GoodDateChooser(LocalDate.now());
+            if (inventory != null) {
+                jCalendarInventory.setDate(inventory.getInventoryDate().toLocalDate());
+            }
+            jCalendarInventory.addDateChangeListener(dateChangeEvent -> {
+                LocalDate newDate = dateChangeEvent.getNewDate();
+                if (newDate != null) {
+                    dateInventory = newDate.atStartOfDay();
+                    jCalendarInventory.setDate(newDate);
+                }
+            });
+        }
+        return jCalendarInventory;
+    }
+
+    private JButton getSaveButton() {
+        saveButton = new JButton(MessageBundle.getMessage("angal.common.save"));
+        saveButton.setMnemonic(MessageBundle.getMnemonic("angal.common.save.btn.key"));
+        saveButton.addActionListener(actionEvent -> {
+            String State = InventoryStatus.draft.toString();
```

Comment:
```suggestion
            String state = InventoryStatus.draft.toString();
```

Reply:
Why load this into a local variable if the value is only used once and that being inside an `if` statement at line 384?